### DISCUSSION TITLE
The spec is not type of list.

### DISCUSSION THREAD

**Discussion by PLPRASU** [StackStorm/st2#5632] [Q&A]
i got error like this when run the workflow - message: The spec is not type of list.

**Comment by amanda11** [comment]
You need to supply more information, with full details of the error, and the workflow you are running, data passed etc. It looks like incompatibility between types expected and what is sent.

**Comment by amanda11** [comment]
@PLPRASU This looks to be duplicate of https://github.com/StackStorm/st2/discussions/5629, if so, can you close this discussion and keep the other one open.
